docs/install/create-project.md

```diff
@@ -155,28 +154,11 @@ Install the latest Yarn 3 version (not the Classic 1.x one).
     corepack enable
     ```
 
-2.  Verify that Yarn v3.x.x is installed and activated.
-
-    ```shell
-    yarn set version 3.x
-    yarn -v
-    ```
-    ```console
-    3.2.3
-    ```
-
-    If you see a version that is not v3.x.x, you can try deleting the {file}`yarn.lock` file, and installing again.
+2.  Set the yarn version to stable.
 
     ```shell
-    rm yarn.lock
-    corepack enable
-    yarn set version 3.x
-    yarn -v
+    yarn set version stable
     ```
-    ```console
-    3.2.3
-    ```
-
 
```

Comment:
The above command installs Yarn 4.1.0. Are we OK with that?

We should also add a step to verify the minimum supported version.

````suggestion

Verify the installed version of Yarn is 3 or later.

```shell
yarn -v
```
```console
4.1.0
```

````